saleor/static/dashboard-next/storybook/Decorator.tsx

```diff
@@ -1,19 +1,16 @@
-import CssBaseline from "@material-ui/core/CssBaseline";
-import MuiThemeProvider from "@material-ui/core/styles/MuiThemeProvider";
 import * as React from "react";
 
 import { Provider as DateProvider } from "../components/Date/DateContext";
 import { FormProvider } from "../components/Form";
 import { MessageManager } from "../components/messages";
+import ThemeProvider from "../components/Theme";
 import { TimezoneProvider } from "../components/Timezone";
-import theme from "../theme";
 
 export const Decorator = storyFn => (
   <FormProvider>
     <DateProvider value={+new Date("2018-08-07T14:30:44+00:00")}>
       <TimezoneProvider value="America/New_York">
-        <MuiThemeProvider theme={theme}>
-          <CssBaseline />
+        <ThemeProvider isDefaultDark={false}>
```

Comment:
The light theme could be used as the default one, but we could also store the information about which theme is enabled in local storage and use that as default then.

Reply:
This is storybook file. Theme is saved in localStorage by default.

Reply:
Right, anyway are you sure it works with localStorage? I've tested that and after closing dashboard, the theme was reset to the default light one.